tests/ide-support/standalonebuilder/assembly.livecodescript

```diff
@@ -0,0 +1,86 @@
+﻿script "StandaloneExtensionsAssembly"
+/*
+Copyright (C) 2018 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+local sSupportStack, sExtensionName
+on TestSetup   
+	breakpoint
+   put "com.livecode.library.assemblytest" into sExtensionName
+   __InstallExtension sExtensionName
+   
+   start using stack "revSBLibrary"
+		
+   set the itemdelimiter to slash
+   local tSupportStack
+   put the filename of me into tSupportStack
+   put "_support.livecodescript" into item -1 of tSupportStack
+   put tSupportStack into sSupportStack
+   start using stack sSupportStack
+end TestSetup
+
+private function __GetExtFolder
+	return TestGetEngineRepositoryPath() & slash & \
+	   "_tests/_build/packaged_extensions/" & sExtensionName
+end __GetExtFolder
+
+private command __InstallExtension
+   local tExtra
+   put "assemblytest-support.lcb" into tExtra["support_files"]
+   breakpoint
+   revIDEExtensionLoad "lcb", sExtensionName, \
+         __GetExtFolder(), "0.0.0", "installed", \
+         "", "true", "assemblytest.lcb", tExtra
+end __InstallExtension
+
+on TestTeardown
+   --StandaloneBuilderCleanUpStandalones
```

Comment:
@livecodeali should this be uncommented

Reply:
I wondered about that - they all seem to be commented out (this is copy-pasted from another one)